dg/config/cluster_credentials_manager.py

```diff
@@ -204,6 +207,36 @@ def get_current_cluster(self) -> Union[AbstractCluster, None]:
 
         return cluster
 
+    def get_current_credentials(self) -> ContextData:
+        """Returns user and current cluster credentials
```

Comment:
Improve inline documentation a bit, by stating where said credentials will be obtained from (file and from current session it seems)